export  async function removeFile(path) {
  try {

  }catch (e) {
    console.log('Error while removing file', e.message)
  }
}